src/services/socket.io.service.js

```diff
@@ -1,22 +1,20 @@
 const io = require('socket.io')();
```

Comment:
socketIo.service.js was a correct naming (there's no such entity as 'io' in project, it's part of the library name), please rename back